pkg/virt-controller/services/template_test.go

```diff
@@ -2137,7 +2137,6 @@ var _ = Describe("Template", func() {
 
 				caps := pod.Spec.Containers[0].SecurityContext.Capabilities
 
-				Expect(caps.Add).To(ContainElement(kubev1.Capability(CAP_NET_ADMIN)), "Expected compute container to be granted NET_ADMIN capability")
 				Expect(caps.Drop).To(ContainElement(kubev1.Capability(CAP_NET_RAW)), "Expected compute container to drop NET_RAW capability")
```

Comment:
If we've abandoned NET_ADMIN, do we still need to explicitly drop NET_RAW (templates.go line 999)?

@AlonaKaplan?

Reply:
These are two different capabilities, I think we explicitly drop them because they come as default.

Reply:
Yes, NET_RAW may come as a default.